packages/backend/checkcaptcha/app.js

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT-0
+
+/**
+ * Event Example:
+ *   {"action": "checkcaptcha", "token": "..."}
+ * 
+ */
+
+const AWS = require('aws-sdk');
+const axios = require('axios');
+
+const { AWS_REGION, SOCKET_TABLE_NAME } = process.env;
+
+const ddb = new AWS.DynamoDB.DocumentClient({ apiVersion: '2012-08-10', region: AWS_REGION });
+const smc = new AWS.SecretsManager({ region: AWS_REGION });
+
+const { getReCaptchaKeys } = require('./utils/getReCaptchaKeys.js');
+
+
+async function updateStatus(connectionId, captchaPassed) {
+  const updateParams = {
+    TableName: SOCKET_TABLE_NAME,
+    Key: { 'connectionId': connectionId },
+    ExpressionAttributeValues: { ':cp': captchaPassed },
+    UpdateExpression: 'set captchaPassed = :cp'
+  };
+
+  try {
+    await ddb.update(updateParams).promise();
+  } catch (err) {
+    throw err;
+  }
+}
+
+async function checkRecaptcha(token) {
+  const {
+    googleApiKey,
+    googleProjectId,
+    reCaptchaKey,
+    reCaptchaAction,
+    reCaptchaThreshold
+  } = await getReCaptchaKeys(smc);
```

Comment:
Update to separate `googleProjectId`, `reCaptchaAction`, and `reCaptchaThreshold`

Reply:
It's now provided as ENV variable to lambda function